GTFS_converter.py

```diff
@@ -0,0 +1,241 @@
+#this script converts GTFS input to spur input
+#input routes.txt, stop_times.txt, stops.txt, trips.txt
+#output components.json, routes.json, tours.json
+#trains.json is not generated as there is no corresponding file in GTFS
+#yards are not generated and to be added by the user

```

Comment:
Possible enhancement or flag we can accept to "append yards?" Let's  perhaps flag it as a separate enhancement to keep it simple.